docs/command_line.md

```diff
@@ -45,31 +46,32 @@ lisa -r ./microsoft/runbook/azure.yml -v location:westus2 -v "gallery_image:Cano
 
 ## run
 
-Run is the default operation. The `run` is optional.
+An optional command since it is the default operation. The following two lines perform the same operation.
 
 ```sh
-lisa run -r ./microsoft/runbook/azure.ym
+lisa run -r ./microsoft/runbook/azure.yml
```

Comment:
Let's put one command in one code snippet. If multiple commands put in the same code snippet, it hints they should be run together.